Buildings/Resources/Scripts/Dymola/Experimental/DHC/Examples/Cooling/DirectUncontrolledETS.mos

```diff
@@ -0,0 +1,33 @@
+// Simulate
+simulateModel("Buildings.Experimental.DHC.Examples.Cooling.DirectUncontrolledETS", tolerance=1e-6, startTime=15724800, stopTime=16329600, method="Dassl", numberOfIntervals=500, resultFile="DirectUncontrolledETS");

```

Comment:
- [ ] Fix simulation errors in this model
- [ ] Ensure all plot variables are appearing correctly

Reply:
Done for Dymola